pkg/chains/formats/slsa/internal/material/material.go

```diff
@@ -56,9 +55,6 @@ func TaskMaterials(ctx context.Context, tro *objects.TaskRunObject) ([]common.Pr
 
 	mats = artifact.AppendMaterials(mats, FromTaskParamsAndResults(ctx, tro)...)
 
-	// add task resources
```

Comment:
of interest

believe we can remove TaskResources as they are no longer present in v1.  For backwards compatibility we will translate v1beta1 TaskResources -> type hinted Results